include/ginkgo/core/log/papi.hpp

```diff
@@ -16,6 +16,8 @@
 #include <iostream>
 #include <map>
 #include <mutex>
+
+
 #include <sde_lib.h>
```

Comment:
actually, I do not get this changes.
do you add new breaks and only run format without format_header?

Reply:
Yes, I forgot the format_header. I will also run this.

Reply:
although I do not know why the format_header change it, it shuold keep everything in `#if` block

Reply:
just go through again the format_header. the thing in #if only raises a warning only. the header is still considered in clang-format with regroup (in format_header)